add required CIDR to the main Prereq section for reducing back-and-forth (#2569)
astronomer/docs: astro/install-gcp-hybrid.md

:::

> To install Astro Hybrid on GCP, Astronomer support creates a Astro cluster in a dedicated GCP account that's hosted and owned by your organization. This ensures that all data remains within your network and allows your organization to manage infrastructure billing.

Reviewer: ```suggestion
To install Astro Hybrid on GCP, Astronomer support creates a cluster in a dedicated GCP account that's hosted and owned by your organization. This ensures that all data remains within your network and allows your organization to manage infrastructure billing.
```

To complete the setup for the installation, you'll:

> - Create an [Astronomer account](#access-astro).

Reviewer: Same edits as in AWS

- Create a new [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
- Activate your Astro data plane by enabling Google Cloud APIs and adding service accounts to your project's IAM.
- Share [the setup information](#provide-setup-information-to-astronomer) with Astronomer.

When you've completed the setup process, Astronomer will create infrastructure within your Google Cloud project to host the resources and Apache Airflow components necessary to deploy DAGs and execute tasks. If you need more than one Astro cluster, contact [Astronomer support](https://cloud.astronomer.io/support).

## Prerequisites

- A [Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) with billing enabled. For security reasons, the install process is not currently supported on a Google Cloud project that has other tooling running in it.

- A user with [Owner permissions](https://cloud.google.com/iam/docs/understanding-roles) in your project.

- [Google Cloud Shell](https://cloud.google.com/shell).

> - CIDR blocks of ranges `/19`, `/20`, `/22`, and `/22` are required for the Astro Data Plane. If you do not have any preferred CIDR block, Astro will provision VPCs using a [default CIDR ranges](#vpc-peering-prerequisites-optional). See [GCP resource reference](resource-reference-azure-hybrid.md) for details.

Reviewer: Same edits as AWS

:::

To install Astro Hybrid on GCP, Astronomer support creates a cluster in a dedicated GCP account that's hosted and owned by your organization. This ensures that all data remains within your network and allows your organization to manage infrastructure billing.

To complete the installation, you'll:

- Create an Astronomer account.
- Create a new Google Cloud project.

> - Enable Google Cloud APIs and add IAM service account that'll be used by Astro.

Reviewer: ```suggestion
- Enable Google Cloud APIs and add an IAM service account that will be used by Astro.
```

    - `servicenetworking.googleapis.com`
    - `dns.googleapis.com`
    - `servicedirectory.googleapis.com`

> - Create a service account `astronomer@astro-remote-mgmt.iam.gserviceaccount.com` that Astro uses to access the data plane.

Reviewer: ```suggestion
- Create a service account called `astronomer@astro-remote-mgmt.iam.gserviceaccount.com` that Astro uses to access the data plane.
```
